src/main/scala/com/microsoft/azure/cosmosdb/spark/partitioner/CosmosDBPartitioner.scala

```diff
@@ -45,53 +45,4 @@ class CosmosDBPartitioner() extends Partitioner[Partition] with CosmosDBLoggingT
       i => CosmosDBPartition(i, partitionKeyRanges.length, partitionKeyRanges(i).getId.toInt, partitionKeyRanges(i).getParents())
     }
   }
-
-  def computePartitions(config: Config,
-                        requiredColumns: Array[String] = Array(),
-                        filters: Array[Filter] = Array(),
-                        hadoopConfig: mutable.Map[String, String]): Array[Partition] = {
-    val adlImport = config.get(CosmosDBConfig.adlAccountFqdn).isDefined
-    var connection: CosmosDBConnection = new CosmosDBConnection(config)
-    connection.reinitializeClient()
-    if (adlImport) {
-      // ADL source
-      val hdfsUtils = new HdfsUtils(hadoopConfig.toMap)
-      val adlConnection: ADLConnection = ADLConnection(config)
-      val adlFiles = adlConnection.getFiles
-      val adlCheckpointPath = config.get(CosmosDBConfig.adlFileCheckpointPath)
-      val adlCosmosDBFileStoreCollection = config.get(CosmosDBConfig.CosmosDBFileStoreCollection)
-      val writingBatchId = config.get[String](CosmosDBConfig.WritingBatchId)
-      val adlMaxFileCount = config.get(CosmosDBConfig.adlMaxFileCount)
-        .getOrElse(CosmosDBConfig.DefaultAdlMaxFileCount.toString)
-        .toInt
-      logDebug(s"The Adl folder has ${adlFiles.size()} files")
-      val partitions = new ListBuffer[ADLFilePartition]
-      var partitionIndex = 0
-      var i = 0
-      while (i < adlFiles.size() && partitionIndex < adlMaxFileCount) {
-        var processed = true
-        if (adlCheckpointPath.isDefined) {
-          processed = ADLConnection.isAdlFileProcessed(hdfsUtils, adlCheckpointPath.get, adlFiles.get(i), writingBatchId.get)
-        } else if (adlCosmosDBFileStoreCollection.isDefined) {
-          val dbName = config.get[String](CosmosDBConfig.Database).get
-          val collectionLink = s"/dbs/$dbName/colls/${adlCosmosDBFileStoreCollection.get}"
-          processed = ADLConnection.isAdlFileProcessed(connection, collectionLink, adlFiles.get(i), writingBatchId.get)
-        }
-        if (!processed) {
-          partitions += ADLFilePartition(partitionIndex, adlFiles.get(i))
-          partitionIndex += 1
-        }
-        i += 1
-      }
-      partitions.toArray
-    } else {
-      // CosmosDB source
-      var query: String = FilterConverter.createQueryString(requiredColumns, filters)
```

Comment:
Why are we removing the computePartitions for the Cosmos DB source as well?

Reply:
I only removed a custom overload used for the ADL operations. This specific overload has only been used in the code path when storing partitions in ADL/File-system. The default overload is still there.

Reply:
Fixed it